### PR TITLE
SpringBoot 

### PR DESCRIPTION
**Pull request by Alejandro-Rojas**
I copied our current program and build it on top of springboot framework. This will help us with dependencies, what we had so far from last sprint still runs.  I tested it in my local branch.  To test it in your local branch it is in a folder called com.dreamteam.database. Just open that folder which has the copied code already in it. I made it separate due to testing purposes. still needs to be tested on your local.

### CODE REVIEW

**Review by JonathanGrant92 — CHANGES_REQUESTED** [review]
com.dreamteam.database/src/test/java/com/dreamteam/database/com/dreamteam/database/OrderHistory.java should be com.dreamteam.database/src/test/java/com/dreamteam/database/OrderHistory.java

Should not repeat "com/dreamteam/database" in the directory path.

**Comment by Alejandro-Rojas** [comment]
fixed file path. Other than that all the apache mail, maven, sql, and other decencies I added that I thought we would need for later on should be working. You can see what decencies are active in the pom file

**Review by PrimerDevs — APPROVED** [review]

**Review by JonathanGrant92 — APPROVED** [review]

**Comment by Alejandro-Rojas** [comment]
testing

File: com.dreamteam.database/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (kevvo15) [inline com.dreamteam.database/src/main/resources/application.properties:1]: What is this file?

**Review by kevvo15 — APPROVED** [review]
It was giving me issues with the import statement of "import java.time*" ... However I believe it was because I did not change the Lamda level to 8 or something like that lol. Regardless, it managed to run after I fixed that so it's good! Once this is merged, I'll branch from the new master to correct a few things for the BuyerEvent

File: com.dreamteam.database/src/main/resources/application.properties (hunk 1 above)
Comment (Alejandro-Rojas) [reply com.dreamteam.database/src/main/resources/application.properties:1]: its part of the webservices class for when we add them later on.

**Review by domschneider — APPROVED** [review]

**Review by jfreda21 — CHANGES_REQUESTED** [review]
I'm going to suggest we don't merge this in.

I pulled down the branch and was only able to get `mvw.cmd` to compile the code, but could not get a successful `mvw.cmd test` or a `mvnw.cmd install` to work.

For starters the `supplier_event.csv` file is missing from the new `com.dreamteam.database` package directory. So this was the first issues.

```
[INFO] Running com.dreamteam.database.DatabaseTest
Resizing the database from 3000 rows to 6000 rows.
Resizing the database from 6000 rows to 12000 rows.
Resizing the database from 12000 rows to 24000 rows.
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.086 s <<< FAILURE! - in com.dreamteam.database.DatabaseTest
[ERROR] supplyTest  Time elapsed: 0.008 s  <<< ERROR!
java.io.FileNotFoundException: Is the data file in the wrong directory?
        at com.dreamteam.database.DatabaseTest.supplyTest(DatabaseTest.java:109)
```
After moving the missing file in place the next set of errors were encountered for the ApplicationTests.java test class:
```
[INFO] Running com.dreamteam.database.DatabaseTest
Resizing the database from 3000 rows to 6000 rows.
Resizing the database from 6000 rows to 12000 rows.
Resizing the database from 12000 rows to 24000 rows.
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.1 s - in com.dreamteam.database.DatabaseTest
```

It appears to me that the SpringBoot configuration is incomplete. Not to mention the introduction of Maven seems to have greatly complicated things for this project, especially not having a consistent IDE project structure or configuration in place.

I think the work on Maven and SpringBoot framework was great from a discovery point of view, but I don't see the value in merging this into Master, it's just greatly complicating things on this project due to the varying levels of experience and knowledge on the team, along with the tight deadlines and minimal requirements of the overall semester focused project.

I think I will add some details about Maven and SpringBoot, and the benefits of using both in our presentation tomorrow, but I don't believe it's something we should be including into our project. 
Let's try to keep it focused on the core requirements being asked of us and just work on those goals without overly complicating the technical implementations.

To recap, ran the following commands with `JAVA_HOME=C:\Program Files\Java\jdk-13.0.2` on my Windows 10 machine with the following results.

1. `mvnw.cmd clean` ✔️ 
2. `mvnw.cmd compile` ✔️
3. `mvnw.cmd test`  ❌ 
4. `mvnw.cmd install` ❌ 

I strongly feel this isn't the right direction for this project, but if the majority of the team wants to move forward with this merged into master I can support that.

**Comment by Alejandro-Rojas** [comment]
Combines alex and Kevins code into this. It ran and worked for me after reviewing and changing their code a bit. 
- added send email using the SendEmailService.class and also Application.class. this to class work together in for the email service from springboot to work. If you want to test the email out go to application.properties in the resources folder underneath the main package. you will see where you can change the email to yours.

**Comment by Alejandro-Rojas** [comment]
please see how the application.class works with the SendEmailService.class and also the application.properties work together  as thats how the springboot framework works. Now compare this code to the one Alex did in the test class. You will find that using the springboot framework requires less coding and less configuration.
